src/LiveQuery/ParseLiveQueryServer.ts

```
@@ -916,6 +916,41 @@ class ParseLiveQueryServer {
+            for (const whereKey of Object.keys(where)) {
+              const rootField = whereKey.split('.')[0];
+              if (protectedFields.includes(whereKey) || protectedFields.includes(rootField)) {
+                throw new Parse.Error(
+                  Parse.Error.OPERATION_FORBIDDEN,
+                  'Permission denied'
+                );
+              }
+            }
+            for (const op of ['$or', '$and', '$nor']) {
+              if (Array.isArray(where[op])) {
+                where[op].forEach((subQuery: any) => checkWhere(subQuery));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)